### PR TITLE
Vignette changes

### PR DESCRIPTION
**Pull request by ChHaeni**
I've gone through the vignette and added some changes.
Some changes are opinion-based, so just add a comment if I should remove specific commits. It's easy for me to rebase and update this PR...

### CODE REVIEW

**Comment by ChHaeni** [comment]
Sorry, accidently pushed to this PR...

File: vignettes/ALFAM2-start.Rmd [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: "Getting started with the ALFAM2 package"
 output: rmarkdown::html_vignette
 vignette: >
-  %\VignetteIndexEntry{Vignette Title}
+  %\VignetteIndexEntry{Getting started with the ALFAM2 package}
```
Comment (sashahafner) [inline vignettes/ALFAM2-start.Rmd:5]: Good catch. 
I think this should be the short name though: ALFAM2-start

**Review by sashahafner — COMMENTED** [review]
Everything looks good to me (except maybe the `print()` addition, but I can remove those later if I decide to).
Thank you!

File: vignettes/ALFAM2-start.Rmd (hunk 1 above)
Comment (sashahafner) [reply vignettes/ALFAM2-start.Rmd:5]: Fixed in 0d69255158bb1039d807b6ba74ebfd9d8a39e5da

**Pull request merged**